Update resuming.mdx (#1930)
wandb/docs: models/runs/resuming.mdx

You can also specify `resume="auto"` to let W&B to automatically try to restart the run on your behalf. However, you will need to ensure that you restart your run from the same directory. See the [Enable runs to automatically resume](#enable-runs-to-automatically-resume) section for more information.

> For all the examples below, replace values enclosed within `<>` with your own. You can find a live demo of a resumed run [here](https://wandb.ai/wandb/resume-run/workspace?nw=nwuserjuliarose).

Reviewer: ```suggestion
For all the examples below, replace values enclosed within `<>` with your own. 
<Tip>[View a live demo of a resumed run](https://wandb.ai/wandb/resume-run/workspace?nw=nwuserjuliarose).</Tip>
```
